logzio/sender.py

```diff
@@ -18,6 +18,7 @@
 
 
 MAX_BULK_SIZE_IN_BYTES = 1 * 1024 * 1024  # 1 MB
+requests_session = requests.Session()
 
```

Comment:
@zt-initech 
Why did you opt to have requests.Session outside of LogzioHandler class instead of a member?

Reply:
I've changed it to me a member